unittests/test_cli.py

```diff
@@ -6,20 +6,20 @@
 import copy
 import itertools
 import os
+import pathlib
```

Comment:
Importing `copy`, `re`, `tempfile` and `unittest` is not needed.

Reply:
`re` is needed here.